tests/quickcheck_test/test_quickcheck.py

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""This module tests the CodeChecker quickcheck feature."""
+
+import glob
+import os
+import subprocess
+import unittest
+
+
+class QuickCheckTestCase(unittest.TestCase):
+    """This class tests the CodeChecker quickcheck feature."""
+
+    @classmethod
+    def setup_class(cls):
+        """Setup the class."""
+        cls.package_dir = \
+            os.path.realpath(os.environ.get('TEST_CODECHECKER_DIR'))
+
+        # Put CodeChecker/bin to PATH so CodeChecker command becomes available.
+        cls.env = os.environ
+        cls.env['PATH'] = \
+            os.path.join(cls.package_dir, 'bin') + ':' + cls.env['PATH']
+
+        cls.test_dir = os.path.join(
+            os.path.dirname(__file__), 'quickcheck_test_files')
+
+        # Change working dir to testfile dir so CodeChecker can be run easily.
+        os.chdir(cls.test_dir)
+
+    def __check_one_file(self, path):
+        """Test quickcheck output on a ".output" file."""
+        with open(path, 'r') as ofile:
+            lines = ofile.readlines()
+
+        options, correct_output = (lines[0].strip(), ''.join(lines[2:]))
+
+        command = 'CodeChecker quickcheck --analyzers clangsa ' + options

```

Comment:
I do not follow. The corresponding build commands must be appended after the 'CodeChecker quickcheck --analyzers clangsa ', but these build commands become available only after 40th line.